core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/StoreCallbacksImpl.scala

```diff
@@ -320,6 +320,9 @@ object StoreCallbacksImpl extends StoreCallbacks with Logging with Serializable
 
   override def shouldStopRecovery(): Boolean =
     MemoryManagerCallback.memoryManager.shouldStopRecovery()
+
+  override def isEnterpriseEdition: Boolean =
+    SnappySession.isEnterpriseEdition
 }
```

Comment:
See the last store PR comment. It is not required since GemFireVersion is a singleton.